firebase-firestore/firebase-firestore.gradle

```diff
@@ -126,7 +126,7 @@ dependencies {
     implementation project(':protolite-well-known-types')
     implementation 'com.google.firebase:firebase-database-collection:18.0.1'
     implementation 'com.google.firebase:firebase-components:17.1.0'
-    implementation 'com.google.firebase:firebase-appcheck-interop:16.1.1'
+    implementation project(':appcheck:firebase-appcheck-interop')
```

Comment:
RTDB also has a dependency on the App Check interop, so will need to be bumped as well.

In addition, App Check integration was also just added to Firebase Auth, so we'll need to add this change to Auth as well once the M130 release goes out. cc @prameshj

Reply:
RTDB included and I have a bug tracking Auth for M131